### PR TITLE
Guides WIP

### PR DESCRIPTION
**Pull request by nameless**
Players:
- [x] ACRE radio's
- [ ] Medical (Advanced)

Makers:
- [ ] ACRE radio's
- [ ] Loadout

### CODE REVIEW

File: guides/players/radios.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+<hr/>
+
+# AN/PRC-343 Usage
+Need a brief here
+### Volume Control (Combine)
+Left clicking the volume knob increases the 343's volume by 10% to a maximum of 100% whilst right clicking will decrease by 10% to a minimum of 20% volume.
+
+### Channel & Block Control (Seek better formatting)
+To change the channel you left click or right click (increase/decrease by 1) the Channel knob to cycle through the 16 channels.
+![alt text](https://cdn.discordapp.com/attachments/386271418305871873/566169026359853069/107410_20190412192712_1.png)
```
Comment (synixebrett) [inline guides/players/radios.md]: Images should go under the brief, before any specifics. This way people have seen the image before the start reading what buttons do.

Comment (synixebrett) [inline guides/players/radios.md]: Images should be added to the repository, not hosted through Discord

File: guides/players/radios.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
+# ACRE Radios
+Need a brief here (This page will go over the basic operation of ACRE radios for Synixe etc)
+## Controls
+Pressing CapsLock will start transmitting through the radio (indicated by a click) until you let go of CapsLock.
```
Comment (synixebrett) [inline guides/players/radios.md]: CapsLock should be \`capslock\` so it is rendered as `capslock`

Comment (synixebrett) [inline guides/players/radios.md]: All controls should be described here, not just caps lock

File: guides/players/radios.md (hunk 1 above)
Comment (nameless) [reply guides/players/radios.md]: No.

File: guides/players/radios.md [hunk 3]
```diff
@@ -1,22 +1,23 @@
...
 ### Volume Control
 Left clicking the volume knob increases the 343's volume by 10% to a maximum of 100% whilst right clicking will decrease by 10% to a minimum of 20% volume.
 
 ### Channel & Block Control
-To change the channel you left click or right click (increase/decrease by 1) the Channel knob to cycle through the 16 channels.
-![alt text](https://cdn.discordapp.com/attachments/386271418305871873/566169026359853069/107410_20190412192712_1.png)
+To change the channel you left click or right click (increase/decrease by 1) the channel knob to cycle through the 16 channels.
+
 To change the block channel number you'll need to open the 343's Handle (as shown in the photo)
 ![alt text](https://cdn.discordapp.com/attachments/386271418305871873/566169598077042719/107410_20190412192722_1_1.png)
```
Comment (synixebrett) [inline guides/players/radios.md]: Move this up

Comment (nameless) [reply guides/players/radios.md]: NO

Comment (synixebrett) [reply guides/players/radios.md]: meh

**Comment by nameless** [comment]
Please review sir

**Comment by synixebrett** [comment]
Looks good as is, just needs to have the last few sections filled in

**Pull request merged**